components/formats-common/src/loci/common/DebugTools.java

```diff
@@ -89,24 +88,43 @@ public static synchronized boolean isEnabled() {
   }
 
   /**
-   * Attempts to enable SLF4J logging via logback or log4j
-   * without an external configuration file.
+   * Sets the root logger level.
+   *
+   * This method will override the root logger level independently of the way
+   * the logging framework has been enabled.
    *
    * @param level A string indicating the desired level
-   *   (i.e.: ALL, DEBUG, ERROR, FATAL, INFO, OFF, TRACE, WARN).
-   * @return {@code} true if logging was successfully enabled
    */
-  public static synchronized boolean enableLogging(String level) {
-    final String[][] toolClasses = new String[][] {
-      new String[] {"loci.common.", "LogbackTools"},
-      new String[] {"loci.common.", "Log4jTools"}
-    };
-
-    for (String[] toolClass : toolClasses) {
+  public static synchronized void setRootLevel(String level) {
+    for (String[] toolClass : TOOLCLASSES) {
       try {
         Class<?> k = Class.forName(toolClass[0] + toolClass[1]);
-        Method m = k.getMethod("enableLogging", String.class);
+        Method m = k.getMethod("setRootLevel", String.class);
         m.invoke(null, level);
+        return;
+      }
+      catch (Throwable t) {
```

Comment:
Interestingly switching  from `Throwable` to `ReflectiveOperationException` broke the ImageJ plugin when using `loci_tools.jar` (as shown by the testing of 5.2.0-m4 /cc @pwalczysko @bramalingam) as it does not ctach the `NoClassDefFoundError` thrown in:

```
java.lang.NoClassDefFoundError: ch/qos/logback/core/Context
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at loci.common.DebugTools.enableLogging(DebugTools.java:132)
	at loci.common.DebugTools.enableLogging(DebugTools.java:159)
	at loci.plugins.LociImporter.run(LociImporter.java:70)
	at ij.IJ.runUserPlugIn(IJ.java:216)
	at ij.IJ.runPlugIn(IJ.java:180)
	at ij.Executer.runCommand(Executer.java:137)
	at ij.Executer.run(Executer.java:66)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: ch.qos.logback.core.Context
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```